descriptors/ModuleDescriptor-template.json

```diff
@@ -275,15 +285,24 @@
           "permissionsRequired": ["orders.pieces.item.put"],
           "modulePermissions": [
             "orders-storage.pieces.item.put",
-            "orders-storage.pieces.item.get"
+            "orders-storage.pieces.item.get",
+            "orders-storage.po-lines.item.get",
+            "orders-storage.purchase-orders.item.get",
+            "acquisitions-units-storage.units.collection.get",
+            "acquisitions-units-storage.memberships.collection.get"
           ]
         },
         {
           "methods": ["DELETE"],
           "pathPattern": "/orders/pieces/{id}",
           "permissionsRequired": ["orders.pieces.item.delete"],
           "modulePermissions": [
-            "orders-storage.pieces.item.delete"
+            "orders-storage.pieces.item.delete",
+            "orders-storage.pieces.item.get",
```

Comment:
Get piece and get PO Line operations can be replaced by one call to receiving history (with piece id as query criteria) which returns `purchaseOrderId`